activerecord/lib/active_record/migration.rb

```diff
@@ -1170,6 +1174,14 @@ def load_migrated
 
     private
 
+    def database_connection
```

Comment:
Sucks we have to do this, but `Migrator` expects `database_connection` to exist in both class and instance methods.